core/schedule/include/schedule/schedule.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <schedule/schedule_day.h>
+#include <database/database.h>
+
+#include <string>
+
+namespace tabetai2::core::schedule {
+
+
+class Schedule {
+public:
+    Schedule(database::Id id, std::string start_date);
+
+    database::Id id() const;
+
+    void add_day(const ScheduleDay& day);
```

Comment:
Since you copy ScheduleDay internally in this function anyway, you could pass by value and use std::move both at the call-sites and internally.